Doc/whatsnew/3.14.rst

```diff
@@ -90,6 +90,32 @@ If you encounter :exc:`NameError`\s or pickling errors coming out of
 New features
 ============
 
+.. _whatsnew314-pep758:
+
+PEP 758 – Allow except and except* expressions without parentheses
+------------------------------------------------------------------
+
+The :keyword:`except` and :keyword:`except* <except_star>` expressions now allow
+parentheses to be omitted when there is only one exception type.
```

Comment:
I missed it in the first place here, but it should be "multiple exception types" here right?